lib/utils.ts

```diff
@@ -1,39 +1,43 @@
-import {browser, ElementFinder} from "protractor";
-import {SaveFullPageMethodOptions} from "webdriver-image-comparison/build/commands/fullPage.interfaces";
+import { ElementFinder, ProtractorBrowser } from 'protractor';
+import { Folders } from 'webdriver-image-comparison/build/base.interface';
 
+import { InstanceData, BaseMethodOptions } from './interfaces';
+
+
```

Comment:
There is also an extra line break here, is that also needed like in the main file?

Reply:
I like this way to split imports and main code. But if you dont, I can remove.